schema-providers-jsonschema/src/test/java/pi2schema/schema/providers/jsonschema/subject/JsonSubjectIdentifierFieldDefinitionTest.java

```diff
@@ -1,11 +1,11 @@
 package pi2schema.schema.providers.jsonschema.subject;
 
-import com.acme.FarmerRegisteredEvent;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import org.instancio.Instancio;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import pi2schema.schema.providers.jsonschema.model.FarmerRegisteredEvent;
```

Comment:
minor:

Is there a way to still keep the test classes in a different package/target folder as of avro/protobuf?